http_service/app.py

```diff
@@ -132,6 +130,39 @@ def is_running(model_name, bug_id):
     return False
 
 
+def get_bug_ids_last_change_time(bug_ids):
+    query = {"id": bug_ids, "include_fields": ["last_change_time", "id"]}
+    bug = bugzilla.get(
```

Comment:
Nit: bugs

Reply:
I wonder if it wouldn't even be faster to do the query directly with requests instead of libmozdata. libmozdata is chunking by 100 bugs at a time simply because it's getting a lot of data for each bug, but if we just get id and last_change_time we might be able to get all of the bugs in the same query.